### PR TITLE
Added add video, update mess menu and add notification functionality to journalist app

### PR DESCRIPTION
**Pull request by aryanagarwal15**
@Rushi98 @tanmaydixit please review and merge.

### CODE REVIEW

**Comment by aryanagarwal15** [comment]
fixes #155

**Comment by Rushi98** [comment]
@aryanagarwal15 the package naming convention based on type of the class (activity, adapter etc.) is not useful. We should instead base it on feature so that one gets clear idea how closely related classes interact and doesn't need to jump 4 packages to add / modify a feature. 

Although the former package naming convention is used in the client app, we'll change it in next big refactor.

Also, package names in small case.

File: journalists/app/src/main/java/com/csatimes/dojmajournalists/Activity/AddVideoActivity.java [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+import java.util.Calendar;
+import java.util.Locale;
+
+import static com.csatimes.dojmajournalists.Utils.FirebaseKeys.VIDEOS;
+import static com.csatimes.dojmajournalists.Utils.Jhc.getFirebaseRef;
+
+public class AddVideoActivity extends AppCompatActivity {
+
+    private final DatabaseReference databaseReference = getFirebaseRef().child(VIDEOS);
+    private String creator_final = "DoJMA";
```
Comment (Rushi98) [inline journalists/app/src/main/java/com/csatimes/dojmajournalists/Activity/AddVideoActivity.java:32]: camelCase

File: journalists/app/src/main/java/com/csatimes/dojmajournalists/Activity/AddVideoActivity.java [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+            video_upload_date.setError(null);
+            final DatePickerDialog dpd;
+            final Calendar c = Calendar.getInstance();
+            final int currentYear = c.get(Calendar.YEAR);
+            final int currentMonth = c.get(Calendar.MONTH);
+            final int currentDate = c.get(Calendar.DAY_OF_MONTH);
+            dpd = new DatePickerDialog(AddVideoActivity.this,
+                    (view, year, month, dayOfMonth) ->
+                            video_upload_date.setText(String.format(Locale.ENGLISH, "%02d-%02d-%04d",
+                                    dayOfMonth, month+1, year)),
```
Comment (Rushi98) [inline journalists/app/src/main/java/com/csatimes/dojmajournalists/Activity/AddVideoActivity.java:117]: See [Simple Date Format](https://developer.android.com/reference/java/text/SimpleDateFormat) for easier way to do this.

Comment (aryanagarwal15) [reply journalists/app/src/main/java/com/csatimes/dojmajournalists/Activity/AddVideoActivity.java:117]: Umm.. ya that makes more sense, but I just copied code from other activites. This implementation is also effective. Not required to change as far as I think.

File: journalists/app/src/main/java/com/csatimes/dojmajournalists/Activity/AddVideoActivity.java (hunk 1 above)
Comment (aryanagarwal15) [reply journalists/app/src/main/java/com/csatimes/dojmajournalists/Activity/AddVideoActivity.java:32]: cool. Will change this.

**Comment by aryanagarwal15** [comment]
@Rushi98 MVC is universally accepted file format for professional projects. I really think that this frameworks makes more sense instead of one folder for one feature. Also moving through filesystem doesn't take many resources. So this makes more sense to me.

**Comment by Rushi98** [comment]
@aryanagarwal15 MVC is a nice thing to have, but we aren't implementing it here. For ex- Our activity classes fetch data as well as manipulate UX.

Moving through file system requires mental context switches. It indeed consumes dev attention, unnecessarily.

**Comment by aryanagarwal15** [comment]
@Rushi98 this update atleast implements some kind of file structure, which was not available before. It will be wasting resources redoing this rn. We can easily move forward with this structure without much/any problem in future. I for one have never seen profession projects using feature wise file structure breakup ever.

If you have someone else who can refactor this now, then by all means go forward, if not please merge this.

**Review by Rushi98 — CHANGES_REQUESTED** [review]
To be refactored

**Comment by Rushi98** [comment]
continued in #157

**Pull request closed without merging**